src/main/webapp/pages/index.html

```diff
@@ -22,7 +22,7 @@
         <h4 class="pull-left" th:text="#{event.title}">_Recent Activities_</h4>
 
         <div class="btn-group pull-right">
-          <a class="btn btn-primary btn-sm" th:href="@{/events/admin}" sec:authorize="hasRole('ROLE_ADMIN')"
+0          <a class="btn btn-primary btn-sm" th:href="@{/events/admin}" sec:authorize="hasRole('ROLE_ADMIN')"
```

Comment:
Remove this 0

Reply:
Removed rogue `0` in 6912dc2504e37e03b7c8e0b2f5823519951844a4